src/app/src/util/util.js

```diff
@@ -407,3 +407,23 @@ export const sortFacilitiesAlphabeticallyByName = data => data
 
         return (a < b) ? -1 : 1;
     });
+
+// Given a list where each item is like { label: 'ABCD', value: 123 }, and
+// a payload which is a list of items like { label: '123', value: 123 },
+// returns a list of items from the payload with their labels replaced with
+// matching items found in the list.
+export const updateListWithLabels = (list, payload) => list
```

Comment:
Cool! How would you feel about adding a test for this? I've been trying to keep this utils file at near 100% coverage in the utils.test.js suite.

Reply:
That's a great idea. I was a little nervous about the tenuousness of this code, adding tests will help with that.

Reply:
Add a test in a fixup and squashed it for simpler review.